packages/elements-core/src/components/TryIt/TryIt.tsx

```diff
@@ -197,81 +214,94 @@ export const TryIt: React.FC<TryItProps> = ({
     }
   };
 
-  const serversSelect = (
-    <Select
-      aria-label="Servers"
-      options={servers.map(server => ({ value: server.description || '' }))}
-      value={chosenServer?.description || ''}
-      size="sm"
-      onChange={(value: React.Key) => {
-        const server = servers.find(server => server.description === value);
-
-        setChosenServer(server);
-      }}
-    />
-  );
-
-  return (
-    <Box rounded="lg" overflowY="hidden">
-      <Panel isCollapsible={false} p={0} className="TryItPanel">
-        <Panel.Titlebar rightComponent={servers.length > 1 ? serversSelect : undefined} bg="canvas-300">
-          <Box role="heading" fontWeight="bold">
-            <Text color={!isDark ? HttpMethodColors[httpOperation.method] : undefined}>
-              {httpOperation.method.toUpperCase()}
-            </Text>
-            <Text ml={2}>{httpOperation.path}</Text>
-          </Box>
-        </Panel.Titlebar>
+  const isOnlySendButton =
+    !httpOperation.security?.length && !allParameters.length && !formDataState.isFormDataBody && !mediaTypeContent;
 
+  const tryItPanelContents = (
+    <>
+      {httpOperation.security?.length ? (
         <TryItAuth
           onChange={setOperationAuthValue}
           operationSecurityScheme={httpOperation.security ?? []}
           value={operationAuthValue}
         />
+      ) : null}
+
+      {allParameters.length > 0 && (
+        <OperationParameters
+          parameters={allParameters}
+          values={parameterValuesWithDefaults}
+          onChangeValue={updateParameterValue}
+          validate={validateParameters}
+        />
+      )}
 
-        {allParameters.length > 0 && (
-          <OperationParameters
-            parameters={allParameters}
-            values={parameterValuesWithDefaults}
-            onChangeValue={updateParameterValue}
-            validate={validateParameters}
-          />
-        )}
+      {formDataState.isFormDataBody ? (
+        <FormDataBody
+          specification={formDataState.bodySpecification}
+          values={bodyParameterValues}
+          onChangeValues={setBodyParameterValues}
+        />
+      ) : mediaTypeContent ? (
+        <RequestBody
+          examples={mediaTypeContent.examples ?? []}
+          requestBody={textRequestBody}
+          onChange={setTextRequestBody}
+        />
+      ) : null}
+
+      <Panel.Content className="SendButtonHolder" mt={4} pt={!isOnlySendButton && !embeddedInMd ? 0 : undefined}>
+        <HStack alignItems="center" spacing={2}>
+          <Button appearance="primary" loading={loading} disabled={loading} onPress={handleSendRequest} size="sm">
+            Send API Request
```

Comment:
![Screenshot 2021-12-07 at 12 46 54](https://user-images.githubusercontent.com/7916523/145029834-990de0ba-0228-4d90-a654-8ce05603d544.png)

Why we are always this name longer and longer? 😂 

We already have problem with responsiveness btw, and this just makes the problem even more apparent.

Reply:
Yeahh, responsive is a whole separate can of worms - any suggestions for right now or do we improve later? It's OK ATM until about 1450px screen width.